src/tenants.ts

```diff
@@ -59,6 +56,7 @@ const TENANTS: Tenants = {
       facilities: ["projections"],
       methodology: ["projections", "practices"],
     },
+    pagesWithRestrictions: ["projections"],
```

Comment:
At least for now, I think we need to have practices in this restriction list as well. Otherwise, when we deploy this change, all current ID users will get the practices page which I don't believe is what we want until we launch next week. After we launch (or even to launch), we can remove practices from this list.

```suggestion
    pagesWithRestrictions: ["projections", "practices"],
```